StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -227,7 +246,117 @@ final class PartnerAuthViewController: UIViewController {
             view.addAndPinSubviewToSafeArea(errorView)
         }
     }
+
+    private func handleAuthSessionCompletion(_ status: String, _ authSession: FinancialConnectionsAuthSession) {
```

Comment:
double checking: the logic of this function didn't change right? just making sure I am not missing double-checking any changed logic

Reply:
yep automatically abstracted out via xcode